external/emulation/emulation/config.py

```diff
@@ -85,9 +86,13 @@ class ModelConfig:
     mask_gscond_identical_cloud: bool = False
     mask_gscond_zero_cloud: bool = False
     enforce_conservative: bool = False
+    mask_gscond_zero_cloud_classifier: bool = False
```

Comment:
With all these gscond specific options we should probably make a GscondModel subclass soon.